airtime_mvc/application/common/FileDataHelper.php

```diff
@@ -68,4 +68,369 @@ public static function getFileExtensionFromMime($mime)
+                  self::resizeImage($rfile, $rfile . '-256.jpg', $ext, 256, 100);
+                  self::resizeImage($rfile, $rfile . '-512.jpg', $ext, 512, 100);
+                  self::resizeImage($rfile, $rfile . '-1024.jpg', $ext, 1024, 100);
+                  self::imgToDataURI($rfile . '-32.jpg', $rfile . '-32');
+                  self::imgToDataURI($rfile . '-64.jpg', $rfile . '-64');
+                  self::imgToDataURI($rfile . '-128.jpg', $rfile . '-128');
+                  self::imgToDataURI($rfile . '-256.jpg', $rfile . '-256');
+              } else {
+                  Logging::info("The file $rfile does not exist");
+              }
+        } else {
+              $get_img = "";
+        }
+        return $get_img;
+    }
+
+    /**
+     * Normalize string
+     *
+     * @param string $string
+     *
+     * @return string Normalized string
+     */
+    private static function normalizePath($string)
+    {
+        static $normal = array (
+          'ƒ' => 'f',
+          'Š' => 'S',
+          'š' => 's',
+          'Ð' => 'Dj',
+          'Ž' => 'Z',
+          'ž' => 'z',
+          'À' => 'A',
+          'Á' => 'A',
+          'Â' => 'A',
+          'Ã' => 'A',
+          'Ä' => 'A',
+          'Å' => 'A',
+          'Æ' => 'E',
+          'Ç' => 'C',
+          'È' => 'E',
+          'É' => 'E',
+          'Ê' => 'E',
+          'Ë' => 'E',
+          'Ì' => 'I',
+          'Í' => 'I',
+          'Î' => 'I',
+          'Ï' => 'I',
+          'Ñ' => 'N',
+          'Ò' => 'O',
+          'Ó' => 'O',
+          'Ô' => 'O',
+          'Õ' => 'O',
+          'Ö' => 'O',
+          'Ø' => 'O',
+          'Ù' => 'U',
+          'Ú' => 'U',
+          'Û' => 'U',
+          'Ü' => 'U',
+          'Ý' => 'Y',
+          'Þ' => 'B',
+          'ß' => 'Ss',
+          'à' => 'a',
+          'á' => 'a',
+          'â' => 'a',
+          'ã' => 'a',
+          'ä' => 'a',
+          'å' => 'a',
+          'æ' => 'e',
+          'ç' => 'c',
+          'è' => 'e',
+          'é' => 'e',
+          'ê' => 'e',
+          'ë' => 'e',
+          'ì' => 'i',
+          'í' => 'i',
+          'î' => 'i',
+          'ï' => 'i',
+          'ð' => 'o',
+          'ñ' => 'n',
+          'ò' => 'o',
+          'ó' => 'o',
+          'ô' => 'o',
+          'õ' => 'o',
+          'ö' => 'o',
+          'ø' => 'o',
+          'ù' => 'u',
+          'ú' => 'u',
+          'û' => 'u',
+          'ü' => 'u',
+          'ý' => 'y',
+          'ý' => 'y',
+          'þ' => 'b',
+          'ÿ' => 'y',
+          ' ' => '_',
+          '(' => '_',
+          ')' => '_',
+          '[' => '_',
+          ']' => '_'
+        );
+        return strtr($string, $normal);
+    }
+
+    /**
+     * Render image
+     * Used in API
+     *
+     * @param string $file
+     * @param string $size
+     */
+    public static function renderImage($file)
+    {
+          $im = @imagecreatefromjpeg($file);
```

Comment:
In my opinion the `@` operator should be purged from PHP itself. What kin of errors does this try to ignore and ca we handle and log them properly in some way?